pennylane/templates/embeddings.py

```diff
@@ -123,8 +124,10 @@ def AngleEmbedding(features, wires, rotation='X'):
         raise ValueError("Rotation has to be `X`, `Y` or `Z`; got {}.".format(rotation))
 
 
-def AmplitudeEmbedding(features, wires):
-    r"""Encodes :math:`2^n` features into the amplitude vector of :math:`n` qubits.
+def AmplitudeEmbedding(features, wires, pad):
```

Comment:
Should we make `pad` true by default?

Reply:
hmm, I think having `pad` false by default will help user better catch their errors.

Reply:
what do you think?

Reply:
I don't mind either way :slightly_smiling_face: Maybe @mariaschuld has thoughts here? My preference is to have a default though, so that it doesn't need to be explicitly provided each time for the user.